config.go

```diff
@@ -8,8 +8,16 @@ import (
 
 var validID *regexp.Regexp = regexp.MustCompile(`\A[A-Za-z0-9._-]*\z`)
 
+type KafkaVersion struct {
```

Comment:
having both `kVersion` and `KafkaVersion` as structs seems like an unnecessary complication; can't you just make one type out of them?